.rubocop.yml

```diff
@@ -1,7 +1,7 @@
 # this file is managed by dry-rb/devtools project
 
 AllCops:
-  TargetRubyVersion: 2.4
+  TargetRubyVersion: "2.7"
```

Comment:
This was needed as pattern matching support was added

Reply:
This file comes from devtools

Reply:
oops! thanks